include/random.h

```diff
@@ -167,13 +167,13 @@ enum RandomTag
     RNG_POISON_POINT,
     RNG_RAMPAGE_TURNS,
     RNG_SECONDARY_EFFECT,
+    RNG_SECONDARY_EFFECT_2,
+    RNG_SECONDARY_EFFECT_3,
     RNG_SLEEP_TURNS,
     RNG_SPEED_TIE,
     RNG_STATIC,
     RNG_STENCH,
     RNG_TRI_ATTACK,
-    RNG_TRIPLE_ARROWS_DEFENSE_DOWN,
-    RNG_TRIPLE_ARROWS_FLINCH,
     RNG_QUICK_DRAW,
```

Comment:
I don't feel confortable with this way of handling multiple secondary effect rolls.
Maybe @mrgriffin can help us here? 👀